tests/test_onnxml_decision_tree_converter.py

```diff
@@ -252,6 +255,24 @@ def test_random_forest_multi_random(self):
         model.fit(X, y)
         self._test_classifier(X, model)
 
+    @unittest.skipIf(
+        not (onnx_ml_tools_installed() and onnx_runtime_installed()), reason="ONNXML test require ONNX, ORT and ONNXMLTOOLS"
+    )
+    @unittest.skipIf(not xgboost_installed(), reason="ONNXML BRANCH_LT test requires XGBoost")
+    def test_xgboost_branch_lt(self):
+        warnings.filterwarnings("ignore")
+        n_features = 28
+        n_total = 100
+        np.random.seed(0)
+        X = np.random.rand(n_total, n_features)
+        X = np.array(X, dtype=np.float32)
+        y = np.random.randint(2, size=n_total)
+
+        # Create XGBoost Model
+        model = xgb.XGBClassifier()
+        model.fit(X, y)
+        self._test_classifier(X, model, conversion_function=convert_xgboost)
```

Comment:
Is this actually required? I think that if we call `convert` directly, internally it should pick `convert_sklearn` or `convert_xgboost` automatically based on the model type.

Reply:
I think it is required, as the issue can only be recreated when converting via onnx-ml. But indeed it seems `convert-model` from `onnxmltools` can be used to auto select the right converter.